### DISCUSSION TITLE
刚在一个群里看到的

### DISCUSSION THREAD

**Discussion by f5723508** [Arondight/Adachi-BOT#682] [General]
刚在一个群里看到的现在可以显示全部角色？
![-233b70b94a9ce6c9](https://user-images.githubusercontent.com/65771380/156353700-6ab3807f-46bf-448c-96d8-52de5007b83c.jpg)

**Comment by mark9804** [comment]
魔改过的，加上了私人 cookie 支持，然后改了前端排版界面

**Reply by Arondight** [reply]
估计就是这个吧 https://github.com/ixCiel/Adachi-BOT

**Reply by mark9804** [reply]
确实是这个，看到对card的改动了
